vividus-extension-selenium/src/main/java/org/vividus/selenium/WebDriverManagerContext.java

```diff
@@ -37,6 +40,12 @@ public <T> T getParameter(WebDriverManagerParameter parameter)
         return initialValueSupplier == null ? testContext.get(key) : testContext.get(key, initialValueSupplier);
     }
 
+    @Override
+    public <T> T get(WebDriverManagerParameter parameter, Supplier<T> initialValueSupplier)
+    {
+        return testContext.get(parameter.getContextKey(), initialValueSupplier);
```

Comment:
I think it makes sense to add check that `parameter.getInitialValueSupplier() == null`, otherwise there could be collisions

Reply:
is it necessary to throw an exception in this case?

Reply:
yes, IllegalArgumentException

Reply:
done